src/main/java/core/basesyntax/figures/RightTriangle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.figures;
+
+public class RightTriangle extends Figure {
+    private int height;
+    private int side;
+
+    public RightTriangle(String color, String name, int height, int side) {
+        super(color, name = "Right triangle");
```

Comment:
In the constructor, the assignment `name = "Right triangle"` is unnecessary since `name` is being set through the `super` call. You should remove `name = "Right triangle"` and just call `super(color, "Right triangle")`.